src/Controls/src/Core/Window/Window.cs

```diff
@@ -198,6 +198,10 @@ public double MinimumHeight
 			return ValidatePositive(coord);
 		}
 
+#if MACCATALYST
+		bool _frameUpdateInProgress = false;
```

Comment:
Is there a way to use the `_batchFrameUpdate` field? That is already being set when the frame is updating.